content/posts/lets-code-contact-form-in-phoenix-part-one.md

```diff
@@ -266,4 +278,8 @@ defmodule MyApp.Support.MessageTest do
 end
 ```
 
-In the next post, I will be adding the module and functions for the `Support` context within the application. This new context will use the `Message` created in this post. Thanks for joining me! 👋
+In the next post, I will be adding the module and functions for the `Support` context within the application. This new context will use the `Message` and `changeset` function created in this post. Thanks for joining me! 👋
```

Comment:
> I will be adding the module and functions for the `Support` context within the application.

What will these modules and functions _do_? I wonder if there's a bit more to tease to get the reader to come back.